guides/doc-Managing_Hosts/topics/proc_registering-a-host.adoc

```diff
@@ -33,83 +33,97 @@ Optional: If the *Registration* feature is not enabled on your {SmartProxy}, ent
 
 .Procedure
 . Navigate to *Hosts* > *Register Host*.
+. Optional: Select a different *Organization*.
+. Optional: Select a different *Location*.
 . Optional: From the *Host Group* list, select the host group to associate the hosts with.
-Fields that inherit value from *Host group*: Operating System, Activation Key(s) and Lifecycle environment.
-. From the *Operating System* list, select the operating system of hosts that you want to register.
-. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through. You must select the internal {SmartProxy} if you do not want to use an external {SmartProxy}.
-. Optional: *Insecure* - This makes the first call insecure, however, during this first call, hosts download the CA file from {Project}. Hosts will use this CA file to connect to {Project} with all future calls making them secure.
+Fields that inherit value from *Host group*: Operating system, Activation Key(s) and Lifecycle environment.
```

Comment:
IBM Style guide says `no` to adding a `(s)`. See "Parenthese to form plural" in chapter 2.

Reply:
Actually, the field in the WebUI is labelled with that precise string, are we sure we want to introduce an inconsistency with the WebUI for the sake of the SG?

Reply:
Perhaps it would be better to markup those field labels with emphasis instead.

Reply:
Oh, sorry, my mistake. I've checked the WebUI and it follows the SG.